lib/features/composer/presentation/composer_bindings.dart

```diff
@@ -186,6 +187,7 @@ class ComposerBindings extends BaseBindings {
     Get.lazyPut(() => DownloadImageAsBase64Interactor(Get.find<ComposerRepository>()));
     Get.lazyPut(() => TransformHtmlEmailContentInteractor(Get.find<EmailRepository>()));
     Get.lazyPut(() => GetAlwaysReadReceiptSettingInteractor(Get.find<ServerSettingsRepository>()));
+    Get.lazyPut(() => UpdateAlwaysReadReceiptSettingInteractor(Get.find<ServerSettingsRepository>()));
```

Comment:
It is not used in `ComposerController`. Please delete it if not needed

Reply:
Done